hw/ip_templates/rstmgr/data/rstmgr.tpldesc.hjson

```diff
@@ -129,6 +129,12 @@
       type: "bool"
       default: "1"
     }
+    {
+      name: "with_cpu"
+      desc: "Generate outputs for a rstmgr that would connect to an instance of Ibex"
```

Comment:
I think this is used the other way around: it generates inputs from ibex for cpu dump.

```suggestion
      desc: "Generate inputs and CSRs to capture a cpu dump."
```